controllers/provisioners/eks/helpers.go

```diff
@@ -125,6 +129,9 @@ func (ctx *EksInstanceGroupContext) GetBasicUserData(clusterName, args string, k
 api-server   = "{{ .ApiEndpoint }}"
 cluster-certificate = "{{ .ClusterCA }}"
 cluster-name = "{{ .ClusterName }}"
+{{- if .MaxPods}}
```

Comment:
Will this default to 0 if not provided? should we have a better default?

Reply:
it will  default to 0 and be falsey